syncs/syncs.go

```diff
@@ -192,6 +192,22 @@ func (m *Map[K, V]) LoadOrStore(key K, value V) (actual V, loaded bool) {
 	return actual, loaded
 }
 
+// LoadOrInit returns the value for the given key if it exists, otherwise f is
```

Comment:
Optional style nit: If we're going to another line anyways, perhaps move "otherwise f is called to construct the value to be set." on the next line anyways. Personally, I'm a fan of https://sembr.org/. YMMV.

Reply:
I format comments with `gq`, but I have arranged this one as such for now.